dashboard/app/models/ability.rb

```diff
@@ -121,6 +121,15 @@ def initialize(user)
         can :read, :pd_workshop_summary_report
         can :read, :pd_teacher_attendance_report
       end
+
+      if user.permission?(UserPermission::WORKSHOP_ADMIN)
+        can :manage, Pd::Workshop
+        can :manage, Pd::Workshop
```

Comment:
Also please add `Pd::WorkshopMaterialOrder`. It's used for an admin-only page [here](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/controllers/pd/workshop_material_orders_controller.rb#L8)